crds/zz_defs.go

```diff
@@ -98,7 +98,7 @@ func operatorsCoreosCom_catalogsourcesYaml() (*asset, error) {
 		return nil, err
 	}
 
-	info := bindataFileInfo{name: "operators.coreos.com_catalogsources.yaml", size: 7987, mode: os.FileMode(420), modTime: time.Unix(1596160005, 0)}
+	info := bindataFileInfo{name: "operators.coreos.com_catalogsources.yaml", size: 8106, mode: os.FileMode(436), modTime: time.Unix(1596638576, 0)}
```

Comment:
This strange/randon change when running `make generate` also happens to another PR in this repo. Wonder what the deal is.